adapters/relevantdigital/relevantdigital.go

```diff
@@ -0,0 +1,265 @@
+		imp.Audio.Ext = nil
+	}
+}
+
+func setTMax(prebidBidRequest *openrtb2.BidRequest, pbsBufferMs int) {
+	timeout := float64(prebidBidRequest.TMax)
+	if timeout <= 0 {
+		timeout = DEFAULT_TIMEOUT
+	}
+	buffer := float64(pbsBufferMs)
+	prebidBidRequest.TMax = int64(math.Min(math.Max(timeout-buffer, buffer), timeout))
+}
+
+func cloneBidRequest(prebidBidRequest *openrtb2.BidRequest) (*openrtb2.BidRequest, error) {
+	jsonRes, err := json.Marshal(prebidBidRequest)
+	if err != nil {
+		return nil, err
+	}
+	var copy openrtb2.BidRequest
+	err = json.Unmarshal(jsonRes, &copy)
+	return &copy, err
+}
+
+func (a *adapter) createBidRequest(prebidBidRequest *openrtb2.BidRequest, params []*openrtb_ext.ExtRelevantDigital) (*openrtb2.BidRequest, error) {
+	bidRequestCopy, err := cloneBidRequest(prebidBidRequest)
+	if err != nil {
+		return nil, err
+	}
+
+	err = patchBidRequestExt(bidRequestCopy, params[0].AccountId)
+	if err != nil {
+		return nil, &errortypes.BadInput{
+			Message: fmt.Sprintf("failed to create bidRequest, error: %s", err),
+		}
+	}
+
+	setTMax(bidRequestCopy, params[0].PbsBufferMs)
+
+	for idx := range bidRequestCopy.Imp {
+		patchBidImpExt(&bidRequestCopy.Imp[idx], params[idx].PlacementId)
+	}
+	return bidRequestCopy, err
+}
+
+func (a *adapter) getImpressionExt(imp *openrtb2.Imp) (*openrtb_ext.ExtRelevantDigital, error) {
+	var bidderExt adapters.ExtImpBidder
+	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
+		return nil, &errortypes.BadInput{
+			Message: "imp.ext not provided",
+		}
+	}
+	relevantExt := openrtb_ext.ExtRelevantDigital{PbsBufferMs: DEFAULT_BUFFER_MS}
+	if err := json.Unmarshal(bidderExt.Bidder, &relevantExt); err != nil {
+		return nil, &errortypes.BadInput{
+			Message: "ext.bidder not provided",
+		}
+	}
+	return &relevantExt, nil
+}
+
+func (a *adapter) buildEndpointURL(params *openrtb_ext.ExtRelevantDigital) (string, error) {
+	params.Host = strings.ReplaceAll(params.Host, "http://", "")
+	params.Host = strings.ReplaceAll(params.Host, "https://", "")
+	params.Host = strings.ReplaceAll(params.Host, RELEVANT_DOMAIN, "")
+
+	endpointParams := macros.EndpointTemplateParams{Host: params.Host}
+	return macros.ResolveMacros(a.endpoint, endpointParams)
+}
+
+func (adapter *adapter) buildAdapterRequest(prebidBidRequest *openrtb2.BidRequest, params []*openrtb_ext.ExtRelevantDigital) (*adapters.RequestData, error) {
+	newBidRequest, err := adapter.createBidRequest(prebidBidRequest, params)
+
+	if err != nil {
+		return nil, err
+	}
+
+	reqJSON, err := json.Marshal(newBidRequest)
+	if err != nil {
+		return nil, err
+	}
+
+	url, err := adapter.buildEndpointURL(params[0])
+	if err != nil {
+		return nil, err
+	}
+
+	return &adapters.RequestData{
+		Method:  "POST",
+		Uri:     url,
+		Body:    reqJSON,
+		Headers: getHeaders(prebidBidRequest),
+	}, nil
+}
+
+func (adapter *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
```

Comment:
Requesting you to please be consistent with the pointer receiver naming. This function should be - 
```
func (a *adapter) MakeReq...